air/src/options.rs

```diff
@@ -406,6 +419,16 @@ impl Default for PartitionOptions {
     }
 }
 
+// BATCHING OPTION
+// ================================================================================================
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(u8)]
+pub enum BatchingType {
+    Linear = 0,
+    Algebraic = 1,
+}
```

Comment:
Question: is `Linear` the term usually used for the non-algebraic approach? I thought it would be more like "independent" or something like that.

Also, let's add doc comments (both for the structure and for the variants) explaining what these do.

Lastly, I would probably implement `Serializable` and `Deserializable` on this structure so that we don't have to leak its internals higher up the chain.

Reply:
I think this is the most common terminology used. Linear signifies the fact that multivariate polynomial with indeterminates the random value symbols has (total) degree one in each variable. Algebraic signifies that the randomness' are derived using an algebraic relation. Algebraic is also called sometimes parametric or curved.
I added a small comment to explain a bit more.

Added the comments and better (de)serialization